res/values/strings.xml

```diff
@@ -568,13 +579,18 @@
     <string name="perm_required_title">Permission required</string>
     <string name="perm_continue">Continue</string>
     <string name="perm_explain_need_for_camera_access">To capture photos and videos, allow Delta Chat to access the camera.</string>
+    <string name="INFOPLIST.NSCameraUsageDescription">This is used to take photos and videos or scan QR codes.</string>
```

Comment:
the INFOPLIST-strings are just updated on ios, https://github.com/deltachat/deltachat-ios/commit/f3d00176497b51a5bb86b8c13e21d17695b2565b. 

i would wait if they are accepted by apple before pushing them to the translators.

Reply:
<string>"Delta Chat uses your contacts to show a list of email addresses you can write to. Delta Chat has no server, your contacts are not sent anywhere.</string>

Delete the quotation mark after the greater. This may cause the application to crash.

Reply:
thanks, good catch, i will fix that. however, en.lproj/InfoPlist.strings has precedence over Info.plist so the wrong string is at least not displayed.